sdk/typescript/api/client.gen.ts

```diff
@@ -2285,21 +2280,25 @@ export class Container extends BaseClient {
   /**
    * Retrieves this container plus a new file written at the given path.
    * @param path Location of the written file (e.g., "/tmp/file.txt").
-   * @param opts.contents Content of the file to write (e.g., "Hello world!").
+   * @param contents Content of the file to write (e.g., "Hello world!").
    * @param opts.permissions Permission given to the written file (e.g., 0600).
    * @param opts.owner A user:group to set for the file.
    *
    * The user and group can either be an ID (1000:1000) or a name (foo:bar).
    *
    * If the group is omitted, it defaults to the same as the user.
    */
-  withNewFile = (path: string, opts?: ContainerWithNewFileOpts): Container => {
```

Comment:
Yes I can confirm this is a breaking change for TypeScript.
The parameter `contents` moved from a struct param into a require unary parameter.